core/src/main/java/io/questdb/network/NetworkUtils.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.network;
+
+import io.questdb.std.Unsafe;
+
+public class NetworkUtils {
+
+    private NetworkUtils() {
+    }
+
+    /**
+     * Return true if a disconnect happened, false otherwise.
+     */
+    public static boolean testConnection(NetworkFacade nf, int fd, long buffer, int bufferSize) {
```

Comment:
can we make this method a part of NetworkFacade? This looks lonely and NF dependant otherwise

Reply:
Yeah, why not. Let me do that.

Reply:
Done in 920136c77fe179cf4ce8efff1bca07b9ca6a4c5e